src/azure-cli/azure/cli/command_modules/databoxedge/generated/commands.py

```diff
@@ -0,0 +1,87 @@
+        client_factory=cf_device)
+    with self.command_group('databoxedge device', databoxedge_device, client_factory=cf_device) as g:
+        g.custom_command('list', 'databoxedge_device_list')
+        g.custom_show_command('show', 'databoxedge_device_show')
+        g.custom_command('create', 'databoxedge_device_create', supports_no_wait=True)
+        g.custom_command('update', 'databoxedge_device_update')
+        g.custom_command('delete', 'databoxedge_device_delete', supports_no_wait=True, confirmation=True)
+        g.custom_command('download-update', 'databoxedge_device_download_update', supports_no_wait=True)
+        g.custom_command('install-update', 'databoxedge_device_install_update', supports_no_wait=True)
+        g.custom_command('scan-for-update', 'databoxedge_device_scan_for_update', supports_no_wait=True)
+        g.custom_command('show-update-summary', 'databoxedge_device_show_update_summary')
+        g.custom_wait_command('wait', 'databoxedge_device_show')
+
+    from ..generated._client_factory import cf_alert
+    databoxedge_alert = CliCommandType(
+        operations_tmpl='azure.mgmt.databoxedge.operations._alerts_operations#AlertsOperations.{}',
+        client_factory=cf_alert)
+    with self.command_group('databoxedge alert', databoxedge_alert, client_factory=cf_alert) as g:
+        g.custom_command('list', 'databoxedge_alert_list')
+        g.custom_show_command('show', 'databoxedge_alert_show')
+
+    from ..generated._client_factory import cf_bandwidth_schedule
+    databoxedge_bandwidth_schedule = CliCommandType(
+        operations_tmpl='azure.mgmt.databoxedge.operations._bandwidth_schedules_operations#BandwidthSchedulesOperations'
+        '.{}',
+        client_factory=cf_bandwidth_schedule)
+    with self.command_group('databoxedge bandwidth-schedule', databoxedge_bandwidth_schedule,
+                            client_factory=cf_bandwidth_schedule) as g:
+        g.custom_command('list', 'databoxedge_bandwidth_schedule_list')
+        g.custom_show_command('show', 'databoxedge_bandwidth_schedule_show')
+        g.custom_command('create', 'databoxedge_bandwidth_schedule_create', supports_no_wait=True)
+        g.custom_command('update', 'databoxedge_bandwidth_schedule_update', supports_no_wait=True)
+        g.custom_command('delete', 'databoxedge_bandwidth_schedule_delete', supports_no_wait=True, confirmation=True)
+        g.custom_wait_command('wait', 'databoxedge_bandwidth_schedule_show')
+
+    from ..generated._client_factory import cf_job
+    databoxedge_job = CliCommandType(
+        operations_tmpl='azure.mgmt.databoxedge.operations._jobs_operations#JobsOperations.{}',
+        client_factory=cf_job)
+    with self.command_group('databoxedge', databoxedge_job, client_factory=cf_job, is_preview=True) as g:
+        g.custom_command('show-job', 'databoxedge_show_job')
+
+    from ..generated._client_factory import cf_node
```

Comment:
could we move all import code to top of the file?

Reply:
This is decided by code gen😂